Sources/Moya/MoyaProvider+Internal.swift

```diff
@@ -4,6 +4,9 @@ import Result
 // MARK: - Method
 
 extension Method {
+    /// A Boolean value determining whether the request supports multipart.
+    /// True: .post, .put, .patch, .connect
```

Comment:
I would avoid giving implementation details in the comment - once you change the method you would have to remember to change the comment too. First line should be sufficient.